### PR TITLE
fix: deprecated kafka-avro

### PR DESCRIPTION
**Pull request by kirrg001**
refs https://jsw.ibm.com/browse/INSTA-1042
refs https://github.com/waldophotos/kafka-avro/issues/120

Tests are already skipped.

### CODE REVIEW

**Review by abhilash-sivan — APPROVED** [review]

File: packages/core/src/tracing/instrumentation/messaging/rdkafka.js [hunk 1]
```diff
@@ -359,3 +364,10 @@ function findInstanaHeaderValues(instanaHeadersAsObject) {
 
   return { level, traceId, longTraceId, parentSpanId };
 }
+
+function logDeprecationKafkaAvroMessage() {
```
Comment (aryamohanan) [inline packages/core/src/tracing/instrumentation/messaging/rdkafka.js:368]: Suggestion for function name `logKafkaAvroDeprecationWarning`

**Review by aryamohanan — APPROVED** [review]
LGTM

**Comment by aryamohanan** [comment]
Qs: Would it be possible to also remove the tests? It seems unnecessary to keep the skipped ones.

File: packages/core/src/tracing/instrumentation/messaging/rdkafka.js (hunk 1 above)
Comment (kirrg001) [reply packages/core/src/tracing/instrumentation/messaging/rdkafka.js:368]: haha yeah you are right, but I am too lazy to change it now :D

**Comment by kirrg001** [comment]
> Would it be possible to also remove the tests?

I decided to keep them in case we need to test something. We do not have a rule when we remove the tests.

**Pull request merged**